apps/keyboard/js/views/emoji_section_view.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+/* global BaseView */
+
+(function(exports) {
+
+function EmojiKeyView() {
+  BaseView.apply(this, arguments);
+}
+
+EmojiKeyView.prototype = Object.create(BaseView.prototype);
+
+
+EmojiKeyView.prototype.render = function() {
+  var contentNode = document.createElement('button');
+  if (this.options.classNames) {
+    contentNode.classList.add.apply(contentNode.classList,
+        this.options.classNames);
+  }
+
+  contentNode.innerHTML = this.options.outputChar;
+
+  this.element = contentNode;
+  this.viewManager.registerView(this.target, this);
+};
+
+function EmojiSectionView(layout, options, viewManager) {
```

Comment:
I don't see why this can't be `SwipableSectionView` and purge the emoji-specific parts.